fbw-a32nx/src/systems/instruments/src/PFD/FMA.tsx

```diff
@@ -1086,15 +1087,15 @@ class B1Cell extends ShowForSecondsComponent<CellProps> {
           y="7.1040988"
         >
           <tspan>{this.verticalText}</tspan>
-          <tspan
-            xml:space="preserve"
-            class={{
-              PulseCyanFill: this.inSpeedProtection,
-              Cyan: this.inSpeedProtection.map(SubscribableMapFunctions.not()),
-            }}
+          <FlashOneHertz
+            bus={this.props.bus}
+            flashDuration={Infinity}
+            flashing={this.inSpeedProtection}
+            className1={'Cyan'}
+            className2={'DimmendCyan Fill'}
           >
-            {this.additionalText}
-          </tspan>
+            <tspan xml:space="preserve">{this.additionalText}</tspan>
```

Comment:
Just noticed an issue here, the V/S selection indication doesn't show because the `FlashOneHertz` element inserts a `<g>` element "inbetween" the text and the tspan. Will have to think about how to fix this

Reply:
I've changed it to modify the class of the children directly, instead of wrapping them in a g element and setting the class on that